subsys/settings/src/settings_priv.h

```diff
@@ -82,8 +85,12 @@ int settings_next_line_ctx(struct line_entry_ctx *entry_ctx);
  * @retval 0 on success,
  * -ERCODE on storage errors
  */
-int settings_line_raw_read(off_t seek, char *out, size_t len_req,
-			   size_t *len_read, void *cb_arg);
+static inline int settings_line_raw_read(off_t seek, char *out, size_t len_req,
+					 size_t *len_read, void *cb_arg)
+{
+	return settings_line_raw_read_until(seek, out, len_req, len_read,
```

Comment:
looks like settings_line_raw_read_until() is static.

Reply:
Oh I missed that. So it is not possible to inline it like that. My bad.

Reply:
@de-nordic and @nvlsianpu so what is the resolution here?

Reply:
@carlescufi Bring it back as it was before my suggestions for change.

Reply:
oh ok. @galak would you mind reverting the suggestion?

Reply:
Not sure I'm following what the issue is with changing `settings_line_raw_read` even if `settings_line_raw_read_until` is `static inline`

Reply:
@nvlsianpu - ping.